### PR TITLE
[GOCART] update to gocart hash f7b34e - important updates for FENGSHA

### PR DESCRIPTION
**Pull request by bbakernoaa**
## PR Author Checklist:

- [x] I have linked PR's from all sub-components involved in section below. 
- [x] I am confirming reviews are completed in ALL sub-component PR's.
- [x] I have run the full RT suite on either Hera/Cheyenne AND have attached the log to this PR below this line:
  - LOG: [RegressionTests_hera.log](https://github.com/ufs-community/ufs-weather-model/files/13133712/RegressionTests_hera.log)
- [x] I have added the list of all failed regression tests to "Anticipated changes" section.
- [x] I have filled out all sections of the template.

### Subcomponents involved:
- [ ] AQM
- [ ] CDEPS
- [ ] CICE
- [ ] CMEPS
- [ ] CMakeModules
- [ ] FV3
- [x] GOCART
- [ ] HYCOM
- [ ] MOM6
- [ ] NOAHMP
- [ ] WW3
- [ ] stochastic_physics
- [ ] none

### Input data
- [x] No changes are expected to input data.
- [ ] Changes are expected to input data:
  - [ ] New input data.
  - [ ] Updated input data.

### Regression Tests:
- [ ] No changes are expected to any regression test.
- [x] Changes are expected to the following tests: atmaero_control_p8 atmaero_control_p8_rad atmaero_control_p8_rad_micro

<details><summary>Tests effected by changes in this PR:</summary>

```
001 cpld_control_p8_mixedmode_intel failed in check_result
007 cpld_control_p8_intel failed in check_result
009 cpld_control_qr_p8_intel failed in check_result
011 cpld_2threads_p8_intel failed in check_result
012 cpld_decomp_p8_intel failed in check_result
013 cpld_mpi_p8_intel failed in check_result
014 cpld_control_ciceC_p8_intel failed in check_result
015 cpld_control_c192_p8_intel failed in check_result
017 cpld_bmark_p8_intel failed in check_result
021 cpld_debug_p8_intel failed in check_result
025 cpld_control_p8_faster_intel failed in check_result
178 atmaero_control_p8_intel failed in check_result
179 atmaero_control_p8_rad_intel failed in check_result
180 atmaero_control_p8_rad_micro_intel failed in check_result
240 cpld_control_p8_gnu failed in check_result
242 cpld_debug_p8_gnu failed in check_result
001 cpld_control_p8_mixedmode_intel failed in check_result
007 cpld_control_p8_intel failed in check_result
009 cpld_control_qr_p8_intel failed in check_result
011 cpld_2threads_p8_intel failed in check_result
012 cpld_decomp_p8_intel failed in check_result
013 cpld_mpi_p8_intel failed in check_result
014 cpld_control_ciceC_p8_intel failed in check_result
015 cpld_control_c192_p8_intel failed in check_result
017 cpld_bmark_p8_intel failed in check_result
021 cpld_debug_p8_intel failed in check_result
025 cpld_control_p8_faster_intel failed in check_result
178 atmaero_control_p8_intel failed in check_result
179 atmaero_control_p8_rad_intel failed in check_result
180 atmaero_control_p8_rad_micro_intel failed in check_result
240 cpld_control_p8_gnu failed in check_result
242 cpld_debug_p8_gnu failed in check_result
```
</details>

### Libraries

- [x] Not Needed
- [ ] Needed
  - [ ] Create separate issue in [JCSDA/spack-stack](https://github.com/JCSDA/spack-stack) asking for update to library. Include library name, library version.
  - [ ] Add issue link from JCSDA/spack-stack following this item 

<details><summary>Code Managers Log</summary>

- [ ] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR.
- [ ] Move new/updated input data on RDHPCS Hera and propagate input data changes to all supported systems.
  - [ ] N/A

### Testing Log:
- RDHPCS
  - [x] Hera
  - [ ] Orion
  - [ ] Hercules
  - [x] Jet
  - [x] Gaea
  - [ ] Cheyenne
- WCOSS2
  - [x] Dogwood/Cactus
  - [x] Acorn
- CI
  - [ ] Completed
- opnReqTest
  - [ ] N/A
  - [ ] Log attached to comment
</details>

### CODE REVIEW

**Comment by bbakernoaa** [comment]
Ran the `atmaero_control_p8` test.  The test failed but did complete. We do expect changes with the result.

**Comment by bbakernoaa** [comment]
@rmontuoro @junwang-noaa If #1920 is going to be delayed I suggest that we push this through as its critical for the EP5 experiments

**Comment by bbakernoaa** [comment]
just re-synced and so this should be ready @junwang-noaa @DeniseWorthen @rmontuoro

**Comment by zach1221** [comment]
@bbakernoaa do you have the RegressionTests_hera.log available for the run against this PR? It looks like 1920 will be held a bit longer and we think it would be best to move forward with this PR.

Edit: hera logs attached now.

**Comment by jkbk2004** [comment]
@BrianCurtis-NOAA FYI: we started working on this pr.

File: GOCART
Comment (lipan-NOAA) [inline GOCART:1]: it looks fine for me

**Review by lipan-NOAA — APPROVED** [review]

**Comment by bbakernoaa** [comment]
> @bbakernoaa do you have the RegressionTests_hera.log available for the run against this PR? It looks like 1920 will be held a bit longer and we think it would be best to move forward with this PR.
> 
> Edit: hera logs attached now.

Thanks.  I did not have them attached.

**Comment by zach1221** [comment]
> Jenkins-ci ORTs failed

Working on this. Instances are not starting up it seems.

**Comment by BrianCurtis-NOAA** [comment]
@bbakernoaa I wanted to warn you about using the same name of the auth branch in your merge branch from your personal fork, we typically recommend a unique name to your branch. Maybe `gocart_fengsha` for example.

It's not going to break anything but has higher potential for issues. No need to change anything, just an FYI.

**Comment by zach1221** [comment]
Testing is complete. @BrianCurtis-NOAA @SadeghTabas-NOAA @jkbk2004 . Final review requests have been sent to you.

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Review by jkbk2004 — APPROVED** [review]

**Comment by bbakernoaa** [comment]
@BrianCurtis-NOAA Understood.  It was a mistake and I just never went back to fix it.  I'll make sure I don't repeat this mistake again.

**Pull request merged**